### DISCUSSION TITLE
[2021.09.06] 오랜 기간 보호한 동물(1)

### DISCUSSION THREAD

**Discussion by pyunji** [pyunji/Hyundai-ITnE-Team_QuadCore#107] [SQL]
[오랜 기간 보호한 동물(1)](https://programmers.co.kr/learn/courses/30/lessons/59044)

**Comment by pyunji** [comment]
[오랜 기간 보호한 동물(1)](https://github.com/pyunji/Hyundai-ITnE-Team_QuadCore/blob/pyunji/pyunji/sql/PR_59044.sql)

**Comment by hyk339** [comment]
[오랜 기간 보호한 동물(1)](https://github.com/hyk339/Hyundai-ITnE-Team_QuadCore/blob/master/khy/sql/PR_59044.sql)

**Comment by pyunji** [comment]
[fetch 관련 참고하면 좋은 문서](https://onggun.tistory.com/362)

**Reply by chaeyeon4u** [reply]
감사해요!ㅎㅎ

**Comment by chaeyeon4u** [comment]
[오랜 기간 보호한 동물(1) solved](https://github.com/pyunji/Hyundai-ITnE-Team_QuadCore/blob/master/ycy/homework/SQL/PR_59044.sql)

**Comment by leechang4** [comment]
[오랜 기간 보호한 동물(1) solved](https://github.com/leechang4/Hyundai-ITnE-Team_QuadCore/blob/master/lch/SQL/PR%2059044.sql)
